lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -251,6 +258,20 @@ system_default_switchport_shutdown:
   config_get_token: ['/^system default switchport shutdown$/']
   # default_value: n/a. This is a user-configurable system default.
 
+vlan_mapping:
+  _exclude: [/N(3|5|6|9)K/]
+  multiple:
+  config_get_token_append: '/^switchport vlan mapping (\d+) (\d+)/'
+  config_set_append: '%s switchport vlan mapping %s %s'
```

Comment:
Reminder that we need to clean these up.. add it to an existing US that makes sense?